### PR TITLE
Added bamboo specific notes

### PR DESCRIPTION
**Pull request by jeeftor**
This is in regards Issue #1107  - some specific notes on Bamboo

Describes how to NOT get stuck in an endless loop and how to have bamboo make commits.

### CODE REVIEW

**Comment by jeeftor** [comment]
I have NO idea why travis is failing here.  I think master branch is off:

I just did (locally) the following:

    git clone https://github.com/fastlane/fastlane.git ftl
    cd ftl
    bundle install
    bundle exec fastlane test

And i'm getting the same error as travis.  `Exit status of command 'rubocop -D' was 1 instead of 0.`  
This is on a 100% clean pull of master from your repo - so something must be off.

**Comment by mikewdeasy** [comment]
@jeeftor Perfect, this looks great.  Thanks for letting me know!

File: docs/Bamboo.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+Build Version bump by fastlane to Version [0.3] Build [8]
+```
+Then you could use the following regex to ignore these commits
+
+```
+^.*Build Version bump by fastlane.*$
+```
+
+
+##Setting repository remote
```
Comment (KrauseFx) [inline docs/Bamboo.md]: Could you add a space after the `#`

File: docs/Bamboo.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#Bamboo Notes
+
+The following notes may be of use if you are using bamboo with fastlane.
+
+
+##Repository setup
```
Comment (KrauseFx) [inline docs/Bamboo.md]: Same on the other headers

**Comment by KrauseFx** [comment]
Looks great, thanks @jeeftor. Could you also modify the `docs/README.md` to link to this `md` file? I added a small note too. 
Don't worry about the CI, we're working on fixing this, your change is fine :+1:

**Comment by jeeftor** [comment]
Changes made as requested.

**Comment by KrauseFx** [comment]
Great, thanks for your contribution @jeeftor :rocket:

**Pull request merged**